app/scripts/metamask-controller.js

```diff
@@ -600,6 +606,43 @@ export default class MetamaskController extends EventEmitter {
         this.platform.showTransactionNotification(txMeta, rpcPrefs);
 
         const { txReceipt } = txMeta;
+
+        // if this is a transferFrom method generated from within the app it may be a collectible transfer transaction
+        // in which case we will want to check and update ownership status of the transferred collectible.
+        if (
+          txMeta.type === TRANSACTION_TYPES.TOKEN_METHOD_TRANSFER_FROM &&
+          txMeta.txParams !== undefined
```

Comment:
Is it possible for `txMeta.type === TRANSACTION_TYPES.TOKEN_METHOD_TRANSFER_FROM` AND **txMeta.txParams === undefined**?

Reply:
It seems unlikely. Added here for extra protection against unexpected scenarios.